test/demo/utils/RemoteChannel.ts

```diff
@@ -77,4 +84,43 @@ export class RemoteChannel {
 
         return createRemoteContextListener(this.executionTarget, id);
     }
+
+    public async addEventListener(eventType: FDC3ChannelEventType): Promise<RemoteChannelEventListener> {
+        const id = await ofBrowser.executeOnWindow(
+            this.executionTarget,
+            function(this: TestWindowContext, channelInstanceId: string, eventType: FDC3ChannelEventType): number {
+                const listenerID = this.channelEventListeners.length;
+                const channel = this.channelTransports[channelInstanceId].channel;
+
+                const handler = (payload: FDC3ChannelEvent) => {
+                    this.receivedChannelEvents.push({listenerID, payload});
+                };
+
+                const unsubscribe = () => {
+                    channel.removeEventListener(eventType as any, handler as any);
+                };
+
+                channel.addEventListener(eventType as any, handler as any);
```

Comment:
What's with the `any`'s here?

Do we not lint this file? These should need linter exceptions to pass `npm run check`..?

Reply:
As discussed